modules/conversion-center/assets/scss/widgets/contact-buttons-var-4.scss

```diff
@@ -84,41 +85,80 @@
 		margin-bottom: var(--e-contact-buttons-contact-gap);
 	}
 
-	.e-contact-buttons__contact-icon-link {
+	.e-contact-buttons__contact-icon-container {
 		align-items: center;
 		background-color: var(--e-contact-buttons-contact-button-bg);
 		border-radius: 50px;
 		display: inline-flex;
 		justify-content: center;
 		transition: $transition-hover;
+	}
+
+	.e-contact-buttons__contact-icon-link {
+		align-items: center;
+		display: flex;
+		flex-direction: row;
+		gap: 14px;
 
 		svg {
 			fill: var(--e-contact-buttons-contact-button-icon);
 		}
 
 		&:hover,
 		&:focus {
-			background-color: var(--e-contact-buttons-contact-button-bg-hover);
-			transition: $transition-hover;
+
+			.e-contact-buttons__contact-icon-container {
+				background-color: var(--e-contact-buttons-contact-button-bg-hover);
+				transition: $transition-hover;
+			}
 
 			svg {
 				fill: var(--e-contact-buttons-contact-button-icon-hover);
 			}
 		}
 
-		&.has-size-small {
+		&.has-size-small .e-contact-buttons__contact-icon-container {
 			height: var(--e-contact-buttons-icon-size-small);
 			width: var(--e-contact-buttons-icon-size-small);
 		}
 
-		&.has-size-medium {
+		&.has-size-medium .e-contact-buttons__contact-icon-container {
 			height: var(--e-contact-buttons-icon-size-medium);
 			width: var(--e-contact-buttons-icon-size-medium);
 		}
 
-		&.has-size-large {
+		&.has-size-large .e-contact-buttons__contact-icon-container {
 			height: var(--e-contact-buttons-icon-size-large);
 			width: var(--e-contact-buttons-icon-size-large);
 		}
 	}
+
+	.e-contact-buttons__contact-tooltip {
+		background-color: var(--e-contact-buttons-tooltip-bg);
+		border-radius: 16px;
+		box-shadow: 2px 4px 4px 0 rgba(0, 0, 0, 0.15);
+		color: var(--e-contact-buttons-tooltip-text);
+		font-size: 13px;
+		line-height: 25px;
+		padding: 4px 14px;
+		position: relative;
+
+		&::after {
+			border-color: transparent transparent transparent var(--e-contact-buttons-tooltip-bg);
+			border-style: solid;
+			border-width: 8px 0 8px 10px;
+			content: "";
+			height: 0;
+			position: absolute;
+			inset-inline-end: -8px;
+			top: 50%;
+			transform: translateY(-50%);
+			width: 0;
+			
+			@if $direction == rtl {
```

Comment:
If possible, please replace with logical properties, such as `border-inline-start-width`. 
I belief that that is fully supported by modern browsers, see https://caniuse.com/?search=border-inline-start-width.

Reply:
Hi @hein-obox thanks for the suggestion, great stuff!
I've also replaced another border that was using the same logic, please see here https://github.com/elementor/elementor/pull/27312/commits/0a5e770e2ed5a8d6a4bddb09c58855f5e4badf7b#diff-37301fb7f13b4f650234f35baf9163eb32daf4964e29c32a0b189f0f927e1537R393